modules/repo/files/commitlint.yml

```diff
@@ -0,0 +1,14 @@
+# Run commitlint on the commit messages in a pull request.
```

Comment:
Can you reference the file in .github/ instead, because that's where we define them?

Reply:
Yea, I might be able to do that, I'll take a look at that.

Reply:
Otherwise you're gonna have to like run a nightly job that syncs changes to .github workflows with these. I do not think we should be changing the place where we have conditioned people to define jobs.